L67-cpp-callback-api.md

```diff
@@ -124,10 +124,10 @@ Besides the content described in the background section, the rationale also incl
+There is more than once mechanism available for implementing the background polling required by the C++ callback API. One has been implemented [on top of the C++ completion queue API](https://github.com/grpc/grpc/pull/25169). In this approach, the callback API uses a number of library-owned threads to call `Next` on an async CQ that is owned by the internal implementation. Currently, the thread count is automatically selected by the library with no user input and is set to half the system's core count, but no less than 2 and no more than 16. This selection is subject to change in the future based on our team's ongoing performance analysis and tuning efforts. Despite being built on the CQ-based async API, the developer using the callback API does not need to consider any of the CQ details (e.g., shutdown, polling, or even the existence of a CQ).
 
-The other implementation is internal to Google; although it uses most of the same code as the open-source project, it additionally incorporates a library that provides threads for efficient event polling and callback execution. This implementation uses the open-source [core callback API](https://github.com/grpc/proposal/pull/181) that is also used by the Python async implementation, but it cannot currently be open-sourced because the library described above is tightly integrated with other Google internals and has a non-portable implementation. It is possible that this approach will be open-sourced at some point.
+It is the gRPC team's intention that that implementation is only a temporary solution. A new structure called an `EventEngine` is being developed to provide the background threads needed for polling, and this sytem is also intended to provide a direct API for application use. This event engine would also allow the direct use of the [core callback API](https://github.com/grpc/proposal/pull/181) that is currently only used by the Python async implementation. If this solution is adopted, there will be a new gRFC for it. This new implementation will not change the callback API at all but rather will only affect its performance. The C++ code for the callback API already has `if` branches in place to support the use of a poller that directly supplies the background threads, so the callback API will naturally layer on top of the `EventEngine` without further development effort.
```

Comment:
Please link to https://github.com/grpc/proposal/blob/master/L68-core-callback-api.md instead of to the PR that created it.